src/Kanbas/Courses/Quizzes/Editor/Details/index.tsx

```diff
@@ -1,11 +1,245 @@
-import { Form } from "react-bootstrap";
+import { Editor } from '@tinymce/tinymce-react';
+import { useEffect, useState } from "react";
+import { Button, Form } from "react-bootstrap";
+import { FaPlus } from 'react-icons/fa';
+import "./index.css"
+import { useNavigate, useParams } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { KanbasState } from '../../../../store';
+import { setQuiz, addQuiz, resetQuiz, updateQuiz, fetchQuizzes } from '../../reducer';
+import { eQuizType } from '../../../../store/enums/eQuizType';
+import { eAssignmentGroup } from '../../../../store/enums/eAssignmentGroup';
+
+function QuizDetailsEditor({ isCreate }: { isCreate: boolean }) {
+    // TODO move this later to env file
+    const API_KEY = "mhgsbn0hpoxglrhp0i5yksi6m4rk8nkutj2fn8qxn9cn7s9i";
+
+    const dispatch = useDispatch();
+    const navigate = useNavigate();
+    const { courseId, quizId } = useParams();
+
+    const quizDetails = useSelector(
+        (state: KanbasState) => state.quizzesReducer.quiz
+    );
+
+    const quizTypes = Object.keys(eQuizType).filter(key => isNaN(Number(key)));
+    const assignmentGroup = Object.keys(eAssignmentGroup).filter(key => isNaN(Number(key)));
+
+    const [value, setValue] = useState("");
```

Comment:
What is value here?

Reply:
This is supposed to be use for grabbing the description from the textbox.

Reply:
Could you please rename it so this becomes more obvious?